### ISSUE TITLE
Tiles persist after deletion

### ISSUE THREAD

**Issue by chiatt** [archesproject/arches-collector#497]
Sometimes tiles persist after deletion - particularly for the map widget

Check with Annabel about specifics if necessary.

**Comment by chiatt** [comment]
perhaps related #457

**Comment by annabelleee** [comment]
Using the Arches Collector on an iPhone (via TestFlight), I have observed the following behavior:
1) For a resource, I drew a point to represent the location.
2) Then I decided to delete the point using the map draw controls
3) Closing out of that screen, the node tile still registers 1 record(s) even though I deleted the data (within the widget) I created.
![IMG_2499](https://user-images.githubusercontent.com/16655095/58128184-e27e0d00-7bcb-11e9-9359-d9d276c0aaca.PNG)
4) Note: I can delete the entire card, and that works fine, but it seems that when you get rid of data at the widget level, it still registers that data has been entered and a record exists.
5) Another example of this is when I entered text  data in the text widget. If I delete the text using the widget it still registers a record.  
![IMG_2500](https://user-images.githubusercontent.com/16655095/58128165-d8f4a500-7bcb-11e9-93d8-81da8def4ca6.PNG)
![IMG_2501](https://user-images.githubusercontent.com/16655095/58128174-dc882c00-7bcb-11e9-83cf-5544780c15e3.PNG)
